service/service.go

```diff
@@ -34,24 +34,25 @@ type Download struct {
 	shutdown            time.Duration
 	healthCheckInterval time.Duration
 	healthCheckRecovery time.Duration
+	healthCheck         *healthcheck.HealthCheck
 }
 
 // DatasetClient is an interface to represent methods called to action on the dataset api
 type DatasetClient interface {
 	GetVersion(ctx context.Context, userAuthToken, serviceAuthToken, downloadServiceAuthToken, collectionID, datasetID, edition, version string) (m dataset.Version, err error)
-	healthcheck.Client
+	// healthcheck.Client
```

Comment:
Commented out code?

Reply:
This was for the old health-check, I forgot to remove it, thanks!